dev_tools/utils/nxdl_utils.py

```diff
@@ -837,7 +808,7 @@ def get_node_at_nxdl_path(
         (class_path, nxdlpath, elist) = get_inherited_nodes(nxdl_path, nx_name, elem)
     except ValueError as value_error:
         if exc:
-            raise NxdlAttributeError(
+            raise NxdlAttributeNotFoundError(
```

Comment:
use RHS for the error

Reply:
The RHS is correct and being used. I will resolve this comment then.